2014-08-28 22:26:37,694 - [INFO] - from play in pool-6-thread-1 
Listening for HTTP on /0:0:0:0:0:0:0:0:9000

2014-08-28 22:26:59,678 - [ERROR] - from application in New I/O worker #1 


! @6jbnb3l0l - Internal server error, for (GET) [/register] ->

play.PlayExceptions$CompilationException: Compilation error[error: cannot find symbol]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:297) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:292) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:292) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:325) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:325) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:325) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:107) ~[play_2.11-2.3.2.jar:2.3.2]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:105) ~[play_2.11-2.3.2.jar:2.3.2]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2014-08-28 22:26:59,780 - [WARN] - from play in New I/O worker #1 
No application found at invoker init

2014-08-28 22:27:02,533 - [ERROR] - from application in New I/O worker #2 


! @6jbnb3p72 - Internal server error, for (GET) [/register] ->

play.PlayExceptions$CompilationException: Compilation error[value Person is not a member of object controllers.routes]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:297) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:292) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:292) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:325) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:325) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:325) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:107) ~[play_2.11-2.3.2.jar:2.3.2]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:105) ~[play_2.11-2.3.2.jar:2.3.2]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2014-08-28 22:27:08,482 - [ERROR] - from application in New I/O worker #2 


! @6jbnb481h - Internal server error, for (GET) [/register] ->

play.PlayExceptions$UnexpectedException: Unexpected exception[IllegalArgumentException: requirement failed: Source file '/home/sulta/Dropbox/git/uis/ui/target/scala-2.11/classes.bak/sbt8230081134195838536.class' does not exist.]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:300) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:292) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:292) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:325) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:325) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:325) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:107) ~[play_2.11-2.3.2.jar:2.3.2]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:105) ~[play_2.11-2.3.2.jar:2.3.2]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]
Caused by: java.lang.IllegalArgumentException: requirement failed: Source file '/home/sulta/Dropbox/git/uis/ui/target/scala-2.11/classes.bak/sbt8230081134195838536.class' does not exist.
	at scala.Predef$.require(Predef.scala:233) ~[scala-library-2.11.1.jar:na]
	at sbt.IO$.copyFile(IO.scala:580) ~[na:na]
	at sbt.IO$.move(IO.scala:764) ~[na:na]
	at sbt.inc.ClassfileManager$$anonfun$transactional$1$$anon$2$$anonfun$complete$5.apply(ClassfileManager.scala:68) ~[na:na]
	at sbt.inc.ClassfileManager$$anonfun$transactional$1$$anon$2$$anonfun$complete$5.apply(ClassfileManager.scala:68) ~[na:na]
	at scala.collection.TraversableLike$WithFilter$$anonfun$foreach$1.apply(TraversableLike.scala:772) ~[scala-library-2.11.1.jar:na]
	at scala.collection.mutable.HashMap$$anonfun$foreach$1.apply(HashMap.scala:98) ~[scala-library-2.11.1.jar:na]
	at scala.collection.mutable.HashMap$$anonfun$foreach$1.apply(HashMap.scala:98) ~[scala-library-2.11.1.jar:na]
	at scala.collection.mutable.HashTable$class.foreachEntry(HashTable.scala:226) ~[scala-library-2.11.1.jar:na]
	at scala.collection.mutable.HashMap.foreachEntry(HashMap.scala:39) ~[scala-library-2.11.1.jar:na]
	at scala.collection.mutable.HashMap.foreach(HashMap.scala:98) ~[scala-library-2.11.1.jar:na]
	at scala.collection.TraversableLike$WithFilter.foreach(TraversableLike.scala:771) ~[scala-library-2.11.1.jar:na]
	at sbt.inc.ClassfileManager$$anonfun$transactional$1$$anon$2.complete(ClassfileManager.scala:68) ~[na:na]
	at sbt.inc.Incremental$.manageClassfiles(Incremental.scala:66) ~[na:na]
	at sbt.inc.Incremental$.compile(Incremental.scala:37) ~[na:na]
	at sbt.inc.IncrementalCompile$.apply(Compile.scala:27) ~[na:na]
	at sbt.compiler.AggressiveCompile.compile2(AggressiveCompile.scala:157) ~[na:na]
	at sbt.compiler.AggressiveCompile.compile1(AggressiveCompile.scala:71) ~[na:na]
	at sbt.compiler.AggressiveCompile.apply(AggressiveCompile.scala:46) ~[na:na]
	at sbt.Compiler$.apply(Compiler.scala:75) ~[na:na]
	at sbt.Compiler$.apply(Compiler.scala:66) ~[na:na]
	at sbt.Defaults$.sbt$Defaults$$compileTaskImpl(Defaults.scala:770) ~[na:na]
	at sbt.Defaults$$anonfun$compileTask$1.apply(Defaults.scala:762) ~[na:na]
	at sbt.Defaults$$anonfun$compileTask$1.apply(Defaults.scala:762) ~[na:na]
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47) ~[scala-library-2.11.1.jar:na]
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42) ~[na:na]
	at sbt.std.Transform$$anon$4.work(System.scala:64) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18) ~[na:na]
	at sbt.Execute.work(Execute.scala:244) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160) ~[na:na]
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30) ~[na:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:262) ~[na:1.7.0_51]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471) ~[na:1.7.0_51]
	at java.util.concurrent.FutureTask.run(FutureTask.java:262) ~[na:1.7.0_51]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145) [na:1.7.0_51]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615) [na:1.7.0_51]
	at java.lang.Thread.run(Thread.java:744) [na:1.7.0_51]

2014-08-28 22:27:10,790 - [ERROR] - from application in New I/O worker #2 


! @6jbnb4bch - Internal server error, for (GET) [/register] ->

play.PlayExceptions$CompilationException: Compilation error[value Person is not a member of object controllers.routes]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:297) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:292) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:292) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:325) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:325) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:325) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:107) ~[play_2.11-2.3.2.jar:2.3.2]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:105) ~[play_2.11-2.3.2.jar:2.3.2]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2014-08-28 22:27:26,522 - [ERROR] - from application in New I/O worker #2 


! @6jbnb58jj - Internal server error, for (GET) [/register] ->

play.PlayExceptions$CompilationException: Compilation error[value Marketing is not a member of object controllers.routes]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:297) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:292) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:292) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:325) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:325) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:325) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:107) ~[play_2.11-2.3.2.jar:2.3.2]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:105) ~[play_2.11-2.3.2.jar:2.3.2]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2014-08-28 22:27:27,841 - [ERROR] - from application in New I/O worker #3 


! @6jbnb5aia - Internal server error, for (GET) [/register] ->

play.PlayExceptions$CompilationException: Compilation error[value Marketing is not a member of object controllers.routes]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:297) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:292) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:292) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:325) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:325) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:325) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:107) ~[play_2.11-2.3.2.jar:2.3.2]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:105) ~[play_2.11-2.3.2.jar:2.3.2]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2014-08-28 22:27:41,963 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-08-28 22:27:42,509 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

